metatron-app/metatron-discovery
# #3729 korean support abbreviations

### Description
Korean support for numerical abbreviations

**Related Issue** : #3729 


### How Has This Been Tested?

1. Create a chart.
2. Check if there is a Korean unit in the unit change.
3. Check that the units are applied normally.

#### Need additional checks?

### Types of changes



### Checklist:


### Additional Context
